CONTRIBUTING.md

```diff
@@ -23,6 +23,41 @@ tox -e integration   # integration tests
 tox                  # runs 'fmt', 'lint', 'static', and 'unit' environments
 ```
 
+## Integration Tests
+
+When running/developing integration tests there will be times when the tests fail. During teardown
```

Comment:
Is there a way you can control the teardown order so the Profiles controller gets deleted last?

Reply:
This happens by default IIUC. 

The order of the test execution is the following:
1. `pytest-operator` will create a new juju model for the integration tests
2. Integration tests will be executed
3. The `deploy_profiles_controller` fixture gets called and deploys the Profile Controller in the above ephemeral namespace
4. Integration tests run
5. During teardown the juju model will be deleted, by `pytest-operator`
6. The Profiles Controller charm will be deleted, alongside the Profiles CRD

But what did you have in mind tho?